harness/determined/pytorch/_pytorch_context.py

```diff
@@ -329,16 +391,15 @@ def _filter_named_parameters(self, optimizer: torch.optim.Optimizer) -> List:
         return [(name, p) for name, p in self._main_model.named_parameters() if p in opt_params]
 
     def _init_device(self) -> torch.device:
-        self.n_gpus = len(self.env.container_gpus)
```

Comment:
noting that this is a "sort of" breaking change.

Like `env`, this was not documented, but also not named in a way that would suggest we thought of it as private.

I'd say collect these changes and put them in a release note, and maybe ask Vish for a breaking change when we release this.

Reply:
Though... bumping the minor version might be overkill for these things.  I could go either way with it.